docs/source/user_guide/runtime-conf.md

```diff
@@ -104,4 +104,5 @@ Example: `test-bucket`
 
 NOTE: If using IBM Cloud Object Storage, you must generate a set of [HMAC Credentials](https://cloud.ibm.com/docs/services/cloud-object-storage/hmac?topic=cloud-object-storage-uhc-hmac-credentials-main) 
 and grant that key at least [Writer](https://cloud.ibm.com/docs/services/cloud-object-storage/iam?topic=cloud-object-storage-iam-bucket-permissions) level privileges.
-Your `access_key_id` and `secret_access_key` will be used as your `cos_username` and `cos_password` respectively.
+Your `access_key_id` and `secret_access_key` will be used as your `cos_username` and `cos_password` respectively, and
+you should also add `cos_secure` as `"True"`.
```

Comment:
The schema should reflect an optional, boolean-valued property named `cos_secure`.

Reply:
this is being handled by #417